css-ui/Overview.bs

```diff
@@ -985,6 +985,36 @@ If the user agent cannot handle any user-defined cursor,
 it must use the cursor keyword at the end of the list.
 Conforming User Agents may, instead of <<url>>, support <<image>> which is a superset.
 
+The UA must support the following image file formats:
+<ul>
+	<li>PNG, as defined in [[!PNG]]
+	<li>SVG, as defined in [[!SVG]], in <a href="http://www.w3.org/TR/svg-integration/#secure-static-mode">secure static mode</a> [[!SVG-INTEGRATION]]
+	<li>any other non-animated image file format that they support
+	for <<image>> in other properties,
+	such as the the 'background-image' property
+</ul>
+
+In addition, the UA should support the following image file formats:
+<ul>
+	<li>SVG, as defined in [[!SVG]], in <a href="http://www.w3.org/TR/svg-integration/#secure-animated-mode">secure animated mode</a> [[!SVG-INTEGRATION]]
+	<li>any other animated image file format that they support
+	for <<image>> in other properties,
+	such as the the 'background-image' property
+</ul>
+
+The UA may also support additional file formats.
+
+Note: At the time of writing this specification,
+the only file formats supported for cursors in all browsers are
```

Comment:
This looks good except "in all browsers" is both something we can't reasonably claim, and dated text - that is, reflects a statement about the present, in a spec that will be read in the future. We should say something like "in common 2015 desktop browsers".